pynecone/compiler/compiler.py

```diff
@@ -208,3 +208,9 @@ def compile_components(components: Set[CustomComponent]):
     # Compile the components.
     code = _compile_components(components)
     return output_path, code
+
+
+def purge_web_pages_dir():
+    """Empty out .web directory.
+    """
+    utils.empty_dir(constants.WEB_PAGES_DIR)
```

Comment:
There's two special files in the pages directory - `._app.js` and `404.js` that are part of the template and not recompiled. I think we need to exclude those two files when purging the dir.